kbase-extension/static/kbase/js/common/cellComponents/fsmBar.js

```diff
@@ -0,0 +1,72 @@
+define(['common/html', 'util/developerMode'], (html, devMode) => {
```

Comment:
This was part of the appCell, but I extracted the code out and made it its own module. It can also be used by other cells that have an FSM and jobs... e.g. the bulk import cell!